src/server/game/Entities/Player/Player.h

```diff
@@ -2338,6 +2361,14 @@ class TC_GAME_API Player : public Unit, public GridObject<Player>
         SceneMgr& GetSceneMgr() { return m_sceneMgr; }
         RestMgr& GetRestMgr() const { return *_restMgr; }
 
+        PlayerPetData* GetPlayerPetDataById(uint32 petId);
```

Comment:
```cpp
        void SendPlayerChoice(ObjectGuid sender, int32 choiceId);
```
was added recently around this point in this file in the master branch in commit https://github.com/TrinityCore/TrinityCore/commit/66621a85e970bf370e4e16c690e0401f4e168c1c#diff-3a912d08e1fdcfa7a92a0bdd3a5883c6 (build fix) and commit https://github.com/TrinityCore/TrinityCore/commit/89c91c271b94f34076be7256378610a2a45c9ed2#diff-3a912d08e1fdcfa7a92a0bdd3a5883c6 - see if you can find a way to incorporate that change.

Reply:
merged before that.
Travis is having problems today ^^

Reply:
Yeah, I noticed. The `master` branch build completed, though. I believe you need to fix the conflict before travis-ci will run again.